source/fem-dem/gls_vans.cc

```diff
@@ -750,9 +773,21 @@ GLSVANSSolver<dim>::assemble_system_matrix()
                                     *this->cell_quadrature,
                                     *this->mapping);
 
+#if DEAL_II_VERSION_GTE(10, 0, 0)
+  max_particle_id = particle_handler.get_max_local_particle_index();
```

Comment:
Add comments here, it is not clear why you need this part.

Reply:
As the reason above, I had to initialize the particle_fluid force to the same dimension as the force in dem. For this the particle_id is defined differently between different version of Deal.II. I will add a comment to explain this.

Reply:
See my comment above on this.

Reply:
done